gplugins/klayout/netlist_graph.py

```diff
@@ -43,9 +43,17 @@ def netlist_to_networkx(
     )
 
     if top_cell:
-        top_circuits = (
-            next(c for c in top_circuits if c.name.casefold() == top_cell.casefold()),
-        )
+        try:
+            top_circuits = (
+                next(
+                    c for c in top_circuits if c.name.casefold() == top_cell.casefold()
+                ),
+            )
+        except StopIteration as e:
+            available_top_cells = [cell.name for cell in top_circuits]
+            raise ValueError(
+                f"{top_cell=!r} not found in the netlist. Available top cells: {available_top_cells!r}"
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Consider simplifying the error message for clarity.

The use of '!r' in the f-string for both variables might make the error message harder to read for end-users. Simplifying the message could improve user experience.

```suggestion
                f"top_cell='{top_cell}' not found in the netlist. Available top cells: {', '.join(available_top_cells)}"
```